include/CL/sycl/math.hpp

```diff
@@ -0,0 +1,165 @@
+#ifndef TRISYCL_SYCL_MATH_HPP
+#define TRISYCL_SYCL_MATH_HPP
+
+/** \file
+    Implement a wrapper around OpenCL math operations
+    Joan.Thibault AT ens-rennes POINT fr
+    This file is distributed under the University of Illinois Open Source
+    License. See LICENSE.TXT for details.
+*/
+
+#include <cmath>
+
+namespace cl {
+namespace sycl {
+#define TRISYCL_MATH_WRAP(FUN) template<typename T> T FUN(T x) {return std::FUN(x);}
```

Comment:
Formatting is wrong around {}.
Lines should have less than 80 characters.
Yes, that means using \ at the end in a macro.
Fortunately editors can deal with this (C-c C-\ in Emacs for example)